.github/actions/setup-node/action.yml

```
@@ -25,7 +25,7 @@ runs:
         path: |
           .cargo
         key: ${{ hashFiles('.github/workflows/cache.yml') }}-${{ runner.os }}-${{ env.OS_ARCH }}-${{ hashFiles('sources/Cargo.lock') }}
-    - run: cargo install --locked --version 0.36.0 cargo-make
+    - run: cargo install --locked --version 0.37.1 cargo-make
```

```suggestion
    - run: cargo install --locked cargo-make
```
I wonder if it's about time we unpinned `cargo-make` and others. It is annoying on the rare occurrence that latest breaks something, but is it as annoying or as frequent as bumping versions?